releases/release-5.1.2.md

```diff
@@ -0,0 +1,151 @@
+    - Fix the issue that the store size in metrics is inaccurate under heavy writing
+    - Fix the potential issue that TiFlash cannot perform GC for the delta data after running for a long time
+    - Fix the issue of wrong results when `new collation` is enabled
+    - Fix the potential panic issue that occurs during lock resolving
+    - Fix a bug that metrics display wrong value
+
++ PD
+
+    - Fix the bug that PD would not fix down-peer in time. [#4077](https://github.com/tikv/pd/issues/4077)
+    - Fix an issue where data is not stored when using max-replicas or location-labels to indirectly update default placement rule [#3886](https://github.com/tikv/pd/issues/3886)
+    - Fix the bug that PD may panic during scaling out TiKV. [#3911](https://github.com/tikv/pd/pull/3911)
+    - Fix the bug that hot region scheduler can not work when the cluster has evict leader scheduler. [#3697](https://github.com/tikv/pd/pull/3697)
+
++ Tools
+
+    + Backup & Restore (BR)
+
+        - Fix the issue that the average speed is not accurate during data backup and restore [#1405](https://github.com/pingcap/br/issues/1405)
+
+    + Dumpling
+
+        - Fix the issue that Dumpling is pending when `show table status` returns incorrect results in some MySQL versions (8.0.3, 8.0.23) [#333](https://github.com/pingcap/dumpling/pull/333)
+
+    + TiCDC
+
+        - Fix a bug that json encoding may cause panic when processing a string type value that is `string` or `[]byte`. [#2783](https://github.com/pingcap/ticdc/pull/2783)
+        - Reduce gRPC window size to avoid OOM [#2725](https://github.com/pingcap/ticdc/pull/2725)
+        - Fix gRPC keepalive error under high memory pressure. [#2720](https://github.com/pingcap/ticdc/pull/2720)
+        - Fix a bug that an unsigned tinyint causes TiCDC to panic. [#2656](https://github.com/pingcap/ticdc/pull/2656)
+        - Fix empty value issue in open protocol. An empty value is no longer output when there is no change in one transaction. [#2612](https://github.com/pingcap/ticdc/issues/2612)
+        - Fix a bug in DDL handling during manual restarts. [#2607](https://github.com/pingcap/ticdc/pull/2607)
+        - Fix the issue that EtcdWorker's snapshot isolation might be wrongly violated when managing the metadata [#2559](https://github.com/pingcap/ticdc/pull/2559)
+        - Fix the issue that multiple processors could write in the same table when re-scheduling this table [#2230](https://github.com/pingcap/ticdc/issues/2230)
+        - Fix the issue of the ErrSchemaStorageTableMiss error and that a changefeed is reset by accident. [#2422](https://github.com/pingcap/ticdc/issues/2422)
+        - Fix the issue that changefeed cannot be removed when the GcTTL Exceeded Error occors [#2391](https://github.com/pingcap/ticdc/issues/2391)
+        - Fix the issue that replicating large tables to `cdclog` fails [#2446](https://github.com/pingcap/ticdc/pull/2446)
```

Comment:
```suggestion
        - Fix a bug that json encoding may cause panic when processing a string type value that is `string` or `[]byte`. [#2758](https://github.com/pingcap/ticdc/issues/2758)
        - Reduce gRPC window size to avoid OOM [#2202](https://github.com/pingcap/ticdc/issues/2202)
        - Fix gRPC keepalive error under high memory pressure. [#2202](https://github.com/pingcap/ticdc/issues/2202)
        - Fix a bug that an unsigned tinyint causes TiCDC to panic. [#2648](https://github.com/pingcap/ticdc/issues/2648)
        - Fix empty value issue in open protocol. An empty value is no longer output when there is no change in one transaction. [#2612](https://github.com/pingcap/ticdc/issues/2612)
        - Fix a bug in DDL handling during manual restarts [#2603](https://github.com/pingcap/ticdc/issues/2603)
        - Fix the issue that EtcdWorker's snapshot isolation might be wrongly violated when managing the metadata [#2559](https://github.com/pingcap/ticdc/pull/2559)
        - Fix the issue that multiple processors could write in the same table when re-scheduling this table [#2230](https://github.com/pingcap/ticdc/issues/2230)
        - Fix the issue of the ErrSchemaStorageTableMiss error and that a changefeed is reset by accident. [#2422](https://github.com/pingcap/ticdc/issues/2422)
        - Fix the issue that changefeed cannot be removed when the GcTTL Exceeded Error occors [#2391](https://github.com/pingcap/ticdc/issues/2391)
        - Fix the issue that replicating large tables to `cdclog` fails [#1259](https://github.com/pingcap/ticdc/issues/1259)[#2424](https://github.com/pingcap/ticdc/issues/2424)

```